### ISSUE TITLE
Consolidate data access code from other repos here

### ISSUE THREAD

**Issue by jtyoung84** [AllenNeuralDynamics/aind-data-access-api#21]
# User story
As a developer, I want a single repository managing our data access layer, so I can easily import the packages where they're needed.

# Acceptance criteria
- [ ] API to list/update data assets in s3
- [ ] API to list/update data assets registered on CodeOcean
- [ ] API to list/update data assets registered to DocumentDB

# Sprint Ready Checklist 
- [ ] 1. Acceptance criteria defined 
- [ ] 2. Team understands acceptance criteria 
- [ ] 3. Team has defined solution / steps to satisfy acceptance criteria 
- [ ] 4. Acceptance criteria is verifiable / testable 
- [ ] 5. External / 3rd Party dependencies identified
- [ ] 6. Ticket is prioritized and sized

# Notes
*Add any helpful notes here.*

**Comment by tmchartrand** [comment]
I'm curious if this breadth is still the intention of this repo ? Specifically, as we're thinking of creating some tools to work with analysis results on s3 and an analysis DocumentDB, I would guess that doing so here might be the way to go rather than a new repo?

**Comment by tmchartrand** [comment]
yeah I think we had agreed on this previously and forgot to close this out. the readme here makes it clear that the scope is currently database-specific at least, so i'll close this.
